insonmnia/hub/router_linux.go

```diff
@@ -136,3 +123,40 @@ func (r *ipvsRouter) Close() error {
 
 	return nil
 }
+
+type ipvsVirtualService struct {
+	vsID    string
+	options *gateway.ServiceOptions
+	gateway *gateway.Gateway
+}
+
+func (s *ipvsVirtualService) ID() string {
+	return s.vsID
+}
+
+func (s *ipvsVirtualService) AddReal(ID string, host string, port uint16) (*route, error) {
```

Comment:
route is a private type but returned from a public method

Reply:
Fixed.